README.md

```diff
@@ -320,6 +323,34 @@ tube_evaluator = TubeEvaluator(annot_filepath, preds_filepath)
+| --gtformat      | Takes in keyword that represents the format that the groundtruth will be in (example: COCO GT JSON format)                                            | python cli.py --gtformat coco                                |         | Available formats: `coco`, `voc`, `imagenet`, `labelme`, `openimg`, `yolo`, `absolute`, `cvat`, `tube`                 |
+| --detformat     | Takes in keyword that represents the format that the detection will be in (either text files or COCO Detection JSON format)                           | python cli.py --detformat coco                               |         | Available inputs: `coco`, `xyrb` (x,y,x2,y2), or `xywh` (x,y,width,height)                                             |
+| --metrics       | Takes in keyword that represents the metric that will be used                                                                                         | python cli.py --metrics voc2012                              |         |                                                                                                                        |
+| --names, -n     | The path of the file that contains the class names for this detection                                                                                 | python cli.py --names voc_classes.txt                        |         | The name file should just be a plaintext file with each class name delimited by a newline.                             |
+| --threshold, -t | Represents the IoU threshold for a given metric                                                                                                       | python cli.py --metrics voc2007 --threshold 0.75             | 0.5     |                                                                                                                        |
+| --prgraph, -pr  | A flag for creating a Precision x Recall graph for the given metric that it is being used in                                                          | python cli.py --metrics voc2007 -pr                          |         | This flag requires the `savepath` argument to be specified                                                             |
+| --savepath, -sp | An optional parameter used to specify where the visual output will be saved to. Required when `-pr` is specified                                      | python cli.py --savepath toyexample/image_out/               |         |                                                                                                                        |
+| --detcoord      | Argument is associated with `detformat` and represents the coordinate system of the detection files. Either absolute (normal coordinates) or relative | python cli.py --detformat xyrb --detcoord rel                |         | Available inputs: `abs`, `rel`                                                                                         |
```

Comment:
why not `--coord_det`?

Reply:
Fixed in https://github.com/rafaelpadilla/review_object_detection_metrics/pull/57/commits/b7cb0e9e74c1f3f27e90eb83a61db86a3798605c